case_study/memory_leaks/leaking_counter_1/README.md

```diff
@@ -4,36 +4,21 @@ This is memory leaking application to test memory debugging tools.
 
 ## Leak debugging lab
 
-Users report that the app consumes more and more memory with every button click.
-Your task is to fix the leak using the tab Memory > Diff, assuming the application is too large to find the issue by
-simply reviewing the code of the button handler.
-
-Setup the lab:
-
-1. Clone DevTools: `git clone git@github.com:flutter/devtools.git`
-
-2. Navigate to the DevTools directory (`cd devtools`) in two console tabs
+1. Setup the lab:
 
-3. In the first console tab:
+    a. Start the latest version of DevTools by following [BETA_TESTING.md](https://github.com/polina-c/devtools/blob/master/BETA_TESTING.md)
 
-    a. Start the app:
+    b. in another console tabs navigate to the DevTools directory (`cd devtools`) and start the app:
 
-    ```
-    cd case_study/memory_leaks/leaking_counter_1
-    flutter run -d macos --profile
-    ```
+        ```
```

Comment:
This code block isn't formatting correctly for some reason.

Reply:
fixed